internal/consts/consts.go

```diff
@@ -244,26 +246,27 @@ const (
 	/*
 		common mount types
 	*/
-	MountTypeDatabase   = "database"
-	MountTypePKI        = "pki"
-	MountTypeAWS        = "aws"
-	MountTypeKMIP       = "kmip"
-	MountTypeRabbitMQ   = "rabbitmq"
-	MountTypeNomad      = "nomad"
-	MountTypeKubernetes = "kubernetes"
-	MountTypeUserpass   = "userpass"
-	MountTypeCert       = "cert"
-	MountTypeGCP        = "gcp"
-	MountTypeKerberos   = "kerberos"
-	MountTypeRadius     = "radius"
-	MountTypeOCI        = "oci"
-	MountTypeOIDC       = "oidc"
-	MountTypeJWT        = "jwt"
-	MountTypeAzure      = "azure"
-	MountTypeGitHub     = "github"
-	MountTypeAD         = "ad"
-	MountTypeConsul     = "consul"
-	MountTypeTerraform  = "terraform"
+	MountTypeDatabase     = "database"
```

Comment:
I would avoid this formatting :)

Reply:
I am pretty sure that this part was me resolving the conflicts after the PR was up. Didn't add this. Is there a way to check?

Reply:
Hmm.. Not entirely sure what you mean here :D 
The one that I added was `MountTypeMongoDBAtlas = "mongodbatlas" `. Not sure why it formatted all of the mount types?

Reply:
Ah my bad! I misread this. No issues here!